Makefile

```diff
@@ -10,6 +10,6 @@ mantle:
 
 install:
 	install -D -t $(DESTDIR)$(PREFIX)/bin coreos-assembler
-	install -D -t $(DESTDIR)$(PREFIX)/libexec/coreos-assembler $$(find src/ -maxdepth 1 -type f)
+	install -D -t $(DESTDIR)$(PREFIX)/libexec/coreos-assembler deps.txt $$(find src/ -maxdepth 1 -type f)
```

Comment:
Hmm, `libexec/` isn't the right place for this though, right? How about `$(PREFIX)/share/coreos-assembler`? I'm sure we'll have other stuff that will fit there eventually.

Reply:
Eh...it feels simpler to me to only have one data directory - are you thinking about architecture independent versus arch-specific executables?  I can't think how that could matter for us, and the trend is really "just use /usr/lib" - look at `/usr/lib/systemd` for example.

Reply:
We could move everything to `/usr/lib/coreos-assembler` instead of `libexec`?

Reply:
Just trying to honour the convention of `libexec/ == executables not meant for users`.
Yeah, `/usr/lib` works for me.